import.Rmd

```diff
@@ -23,18 +25,18 @@ The common link between all these packages is they all aim to take your data and
 
 ## Flat files
 
-There are many ways to read flat files into R. If you've be using R for a while, you might be familiar with `read.csv()`, `read.fwf()` and friends. We're not going to use these base functions. Instead we're going to use `read_csv()`, `read_fwf()`, and friends from the readr package. Because:
+There are many ways to read flat files into R. If you've be using R for a while, you might be familiar with `read.csv()`, `read.fwf()` and friends. We're not going to use these functions from base R. Instead we're going to use `read_csv()`, `read_fwf()`, and friends from the readr package because:
```

Comment:
'base R' sounds a bit better to my ear than just 'base', but imagine that's quite subjective :)